src/pages/user-list/user-list.tsx

```diff
@@ -28,7 +32,7 @@ export const UserList = () => {
   };
 
   useEffect(() => {
-    getUSersData();
+    getUsersData();
```

Comment:
I think that maybe the base branch of this PR is not the correct one, tomorrow we can see it :)